lib/src/integrationTest/kotlin/com/lemonappdev/konsist/core/util/LocationHelperTest.kt

```diff
@@ -5,6 +5,28 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 class LocationHelperTest {
+    @Test
+    fun `reside in any package`() {
+        // given
+        val desiredPackage1 = "com.domain.update.usecase"
+        val desiredPackage2 = "com.domain"
+        val desiredPackage3 = "org.example.project"
+        val sut = LocationHelper
+
+        // then
+        assertSoftly(sut) {
+            resideInLocation("..", desiredPackage1) shouldBeEqualTo true
+            resideInLocation("..", desiredPackage2) shouldBeEqualTo true
+            resideInLocation("..", desiredPackage3) shouldBeEqualTo true
+            resideInLocation(".", desiredPackage1) shouldBeEqualTo false
+            resideInLocation(".", desiredPackage2) shouldBeEqualTo false
+            resideInLocation(".", desiredPackage3) shouldBeEqualTo false
+            resideInLocation("", desiredPackage1) shouldBeEqualTo false
```

Comment:
Since an empty string and string with `"."` are incorrect values we could forbid the passing of empty strings to avoid potential mistakes made by developers.

```kotlin
requireNot(string.isEmpty()) { "Package name is empty" }

require(string != ".") { "Incorrect package selector format: $pckagee" }
```